### PR TITLE
Layout+Spring

### PR DESCRIPTION
**Pull request by fredthuo**
<dependency>
            <groupId>com.atlassian.jira</groupId>
            <artifactId>jira-rest-java-client-core</artifactId>
            <version>5.1.0</version>
        </dependency>

        <dependency>
            <groupId>io.atlassian.fugue</groupId>
            <artifactId>fugue</artifactId>
            <version>4.5.1</version>
        </dependency>

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -91,93 +137,73 @@
...
-              <item name="main" type="462c0819:PsiDirectoryNode" />
-              <item name="java" type="462c0819:PsiDirectoryNode" />
+              <item name="IdeaProjects" type="3d21c010:ScopeViewTreeModel$ProjectNode" />
+              <item name="IdeaProjects" type="2674bda8:ScopeViewTreeModel$GroupNode" />
             </path>
             <path>
-              <item name="Parking" type="b2602c69:ProjectViewProjectNode" />
-              <item name="Parking" type="462c0819:PsiDirectoryNode" />
-              <item name="target" type="462c0819:PsiDirectoryNode" />
+              <item name="IdeaProjects" type="3d21c010:ScopeViewTreeModel$ProjectNode" />
```
Comment (leahnjoroge) [inline .idea/workspace.xml:159]: what is idea projects do?

File: src/main/java/jira/HelloWorld.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package jira;
+
+import com.atlassian.connect.spring.AtlassianHostUser;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
```
Comment (leahnjoroge) [inline src/main/java/jira/HelloWorld.java:10]: what is this class for?

File: pom.xml [hunk 3]
```diff
@@ -96,6 +98,25 @@
             <artifactId>spring-webmvc</artifactId>
             <version>${spring.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.atlassian.connect</groupId>
+            <artifactId>atlassian-connect-spring-boot-starter</artifactId>
+            <version>1.4.2</version>
+        </dependency>
```
Comment (leahnjoroge) [inline pom.xml:106]: is the maven swagger in this branch?

File: .idea/workspace.xml (hunk 1 above)
Comment (fredthuo) [reply .idea/workspace.xml:159]: That automatically generated itself, I have to do more research on what it does

File: src/main/java/jira/HelloWorld.java (hunk 2 above)
Comment (fredthuo) [reply src/main/java/jira/HelloWorld.java:10]: I was testing if the project builds and runs with no errors

File: pom.xml (hunk 3 above)
Comment (fredthuo) [reply pom.xml:106]: For some reason it's not, don't merge this one I will create another pull request

**Pull request closed without merging**